services/app-web/src/pages/SubmissionRevisionSummary/SubmissionRevisionSummaryV2.tsx

```diff
@@ -77,35 +84,28 @@ export const SubmissionRevisionSummaryV2 = (): React.ReactElement => {
     }
 
     if (
-        !contract ||
-        contract.packageSubmissions.length <= Number(revisionVersion)
```

Comment:
similar issue to https://github.com/Enterprise-CMCS/managed-care-review/pull/2441/files#r1597233952